src/command/dev/mod.rs

```diff
@@ -47,27 +47,37 @@ pub struct DevOpts {
     #[clap(flatten)]
     pub schema_opts: SchemaOpts,
 
-    #[clap(long)]
-    pub name: Option<String>,
+    #[clap(flatten)]
+    pub subgraph_opt: OptionalSubgraphOpt,
 }
 
 #[derive(Debug, Parser, Serialize)]
 pub struct SchemaOpts {
-    /// Url of a running subgraph that a graph router can send operations to
-    /// (often a localhost endpoint).
-    #[clap(long)]
+    /// Url of a running subgraph that a graph router can send operations to, (i.e., http://localhost:4001).
+    ///
+    /// If you pass a `--command` argument and no `--url` argument,
+    /// `rover dev` will attempt to detect the endpoint by doing a scan of your ports. If you find
+    /// this takes too long or is unable to detect your GraphQL server, pass the `--url` argument
+    /// to skip the auto-detection step.
```

Comment:
```suggestion
    /// If you pass the `--command` option but not the `--url` option,
    /// `rover dev` attempts to detect your subgraph's endpoint by scanning your ports. To speed up
    /// startup and avoid a failed detection, we recommend always passing the `--url` option.
```

Reply:
What happens if you provide _neither_ `--command` nor `--url`? Is that allowed? I assume that if so, the same auto-detection occurs. If that's the case, we should tweak this language somewhat.

Reply:
If you don't provide either, we will run the auto-detection, if it can't find anything, it will prompt you for the command to start your GraphQL server (though perhaps it should instead ask you if you have a URL _or_ a command first, and _then_ ask the appropriate question)

Reply:
Cool, tweaked a bit